### ISSUE TITLE
How to let tensorflow uses the new installed protobuf?

### ISSUE THREAD

**Issue by haibarasiao** [tensorflow/tensorflow#3923]
When I installed tensorflow with "pip install",I meet the "A protocol message was rejected because it was too big (more than 67108864 bytes) ",and I uninstall protobuf with pip,then download a new protobuf,install it with changing 256 << 20 in coded_stream.h as "Readme" said,but the same problem occurred,it seems that tensorflow hasn't used the new protobuf,how could I deal with this problem?

**Comment by yaroslavvb** [comment]
Sounds like a protobuf version mismatch, do you get the same problem if you do a fresh install in virtualenv using the [official instructions](https://www.tensorflow.org/versions/r0.10/get_started/os_setup.html)?

**Comment by haibarasiao** [comment]
@yaroslavvb sure I do ,but it didn't work too. I also tried to upgrade the > 64 version limited as said in " https://www.tensorflow.org/versions/r0.10/get_started/os_setup.html#protobuf-library-related-issues"
,but after I upgraded , I can't import tensorflow , reason is segment fault. My tensorflow is installed by pip,
not from source.Could some one can give me a  > 64 limited protobuf version that can be upgrade by pip?Thanks!
